packages/terra-i18n/lib/translations/en-US.js

```diff
@@ -11,7 +11,12 @@ function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { de
 (0, _reactIntl.addLocaleData)(_en2.default);
 
 var messages = {
-  'Terra.ajax.error': 'This content failed to load.'
+  'Terra.ajax.error': 'This content failed to load.',
+  'Terra.demographicsBanner.dateOfBirth': 'DOB: ',
```

Comment:
I fully agree with the idea mentioned in that blog. It's a different case in here. In here, this translation is just the translation string value for the label. A good example in demographics banner is in this link: https://github.com/cerner/terra-core/blob/Terra-Demographics-Banner/packages/terra-demographics-banner/src/translations/en-US.js#L2
```
'DOB: ' + DOBValueComponent
```